app/ts/components/TransactionPage/TransactionDetails.tsx

```diff
@@ -178,5 +178,7 @@ const TokenAmount = ({ receipt, addFavoriteStore }: TokenAmountProps) => {
 			const amount = formatUnits(tokenValue, decimals)
 			addFavoriteStore.value = { ...addFavoriteStore.peek(), amount, token: query.value.value }
 			return <Info label='Amount' value={`${amount} ${symbol}`} />
+		default:
+			return <></>
```

Comment:
shouldn't this be error?

Reply:
right, it didn't warn tho. removed as it's unnecessary as all cases are covered